### ISSUE TITLE
Show What Symlink Points To

### ISSUE THREAD

**Issue by JosephTLyons** [nushell/nushell#1223]
**Is your feature request related to a problem? Please describe.**
When using bash or zsh on Mac, running `ls` in a folder that contains symlinks will display the symlinks as such: 

```
.zprofile -> /Users/josephlyons/.dotfiles/files/.zprofile
```

nushell does not seem to show where the symlink points, which seems to be a vital piece of information for a symlink.

**Describe the solution you'd like**
It would be great if nushell could (somehow) indicate what file the symlink is pointing to is.

**Describe alternatives you've considered**
I have no alternatives to this, other than using `^ls -la`.

If this is already possible or there is an additional command I missed, I apologize, and would appreciate hearing about it.  Thanks.

**Comment by JosephTLyons** [comment]
Any idea of this is a desired feature or not?

**Comment by thegedge** [comment]
Yeah, I think this feature makes a lot of sense to implement 👍

**Comment by JosephTLyons** [comment]
> Yeah, I think this feature makes a lot of sense to implement 👍

I can't be sure it is something I'm capable of implementing, but I could give it a try.  I feel there should be some discussion on how it should look visually.  I know Nu has mostly set columns (name, type, size, created, accessed, and modified) for items and none of them seem to be good candidates for some sort of indication of where the symlink points.  I'm not sure where would be the best place to display this information.  Any suggestions?

**Comment by thegedge** [comment]
The `name` column makes a lot of sense, especially if it's being asked for explicitly (like `ls --follow-symlinks`). If you do that, though, you lose the original name. That would make it hard to do something with both, so perhaps something like `--with-symlink-targets` that has a `target` column? There's a cost to resolving these, so I think it makes sense to be an option.

That's my five cents, but I'll think about it a tiny bit more. Perhaps others have some thoughts?

> I can't be sure it is something I'm capable of implementing, but I could give it a try

If you need any help, join us on Discord. Lots of helpful people in `#developing` that can help you out 🙂

**Comment by JosephTLyons** [comment]
> The `name` column makes a lot of sense, especially if it's being asked for explicitly (like `ls --follow-symlinks`). If you do that, though, you lose the original name. That would make it hard to do something with both, so perhaps something like `--with-symlink-targets` that has a `target` column? There's a cost to resolving these, so I think it makes sense to be an option.
> 
> That's my five cents, but I'll think about it a tiny bit more. Perhaps others have some thoughts?
> 
> > I can't be sure it is something I'm capable of implementing, but I could give it a try
> 
> If you need any help, join us on Discord. Lots of helpful people in `#developing` that can help you out 🙂

I think the idea of a new column of "target" that shows up when using `--with-symlink-targets` seems like a good solution.  I guess another question would be, is it possible to configure new to automatically use certain options when certain commands are called?  I guess this would be an alias

**Comment by JosephTLyons** [comment]
I decided to toy around with the idea here, knowing that the actual implementation has not been settled yet, but I wanted to share what I've noticed.  

1) One of the issues is that long links to the files can often cause the lines to wrap.   I'm not sure if this is an actual issue, since I'm not sure if there is anything that can be done about it, but it doesn't look too nice IMHO.  I'm on a 16 inch MacBook Pro, so it could be worse or better for others.  

2) I'm not sure which position in the table the column should be in or what it's actual name should be.   

I just sort of pieced this together for the sake of seeing how it would look, but it is lacking a few things and would require more research to see how things are officially done in this project.

Here's sample output of `ls --full` (the actual option for this is something I haven't done yet, I just added it to `--full` in order to see how it looked):

<img width="1626" alt="Screen Shot 2020-01-27 at 1 33 50 AM" src="https://user-images.githubusercontent.com/19867440/73154709-34096080-40a6-11ea-973b-94b20b88e92b.png">
